components/datetime/src/time_zone.rs

```diff
@@ -166,6 +166,7 @@ pub(super) enum TimeZoneFormatterUnit {
     LocalizedOffsetLong,
     LocalizedOffsetShort,
     Iso8601(Iso8601Format),
+    #[allow(dead_code)]
     ExemplarCity,
```

Comment:
issue: remove this and `ExemplarCityFormat`, because that is what's actually blocking